deployment/aws/cdk/app.py

```diff
@@ -170,7 +170,9 @@ def __init__(
     perms.append(
         iam.PolicyStatement(
             actions=["s3:GetObject", "s3:HeadObject"],
-            resources=[f"arn:aws:s3:::{bucket}*" for bucket in settings.buckets],
+            resources=[
```

Comment:
this should have a check for if `bucket == "*"` and not append the key in that case.

Reply:
> otherwise you end up with */*

I'm not sure to understand, reading the documentation is seems to be that `*/*` is valid

Reply:
It's valid, but it's just redundant to `*`, but maybe it's fine if it makes the code simpler.